ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/result-sort/result-sort.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { hot } from 'react-hot-loader'
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import MarionetteRegionContainer from '../../container/marionette-region-container'
+import { WithBackboneProps } from '../../container/backbone-container'
+import { Button, buttonTypeEnum } from '../button'
+
+const SortItemCollectionView = require('../../../component/sort/sort.view.js')
+const Backbone = require('backbone')
+const user = require('../../../component/singletons/user-instance.js')
+
+type Props = {
+  collection: Backbone.Collection<Backbone.Model>
+} & WithBackboneProps
+
+type State = {
+  collection: Backbone.Collection<Backbone.Model>
+}
+
+const Root = styled.div`
+  display: block;
+  padding: 10px;
+  min-width: 500px;
+
+  .editor-footer {
+    padding-top: 10px;
+  }
+
+  .footer-remove {
+    display: inline-block;
+    width: 100%;
+  }
+
+  .footer-remove-has-sort {
+    display: inline-block;
+    width: 50%;
+  }
+`
+
+export default hot(module)(
```

Comment:
`export default hot(module)(ResultSort)` at the bottom of the file instead

Reply:
the majority of the react components follow the structure here; imo this is more of a ✏️

Reply:
did it anyway 👍